app/admin/settings/shipping/page.tsx

```
@@ -0,0 +1,362 @@
+                {availableCarriers.map((carrier) => (
+                  <option key={carrier.value} value={carrier.value}>
+                    {carrier.label}
+                  </option>
+                ))}
+              </select>
+            </div>
+
+            <div className="mt-4">
+              <label className="block text-sm font-medium text-slate-700">
+                Enabled carriers
+              </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)